cmd/skopeo/signing.go

```diff
@@ -67,15 +67,21 @@ func (opts *standaloneSignOptions) run(args []string, stdout io.Writer) error {
 }
 
 type standaloneVerifyOptions struct {
+	truststore string
 }
 
 func standaloneVerifyCmd() *cobra.Command {
 	opts := standaloneVerifyOptions{}
 	cmd := &cobra.Command{
 		Use:   "standalone-verify MANIFEST DOCKER-REFERENCE KEY-FINGERPRINT SIGNATURE",
 		Short: "Verify a signature using local files",
-		RunE:  commandAction(opts.run),
+		Long: `Verify a signature using local files
+
+KEY-FINGERPRINT can be an exact fingerprint, or "any" if you trust all the keys in the trust store.`,
+		RunE: commandAction(opts.run),
 	}
+	flags := cmd.Flags()
+	flags.StringVar(&opts.truststore, "truststore", "", "Trust store of public keys. Defaults to using local gpg keys.")
```

Comment:
This parameter is just a file that contains public keys, so `key-file`, `keys` or something like that?

Reply:
I think trust store is more specific, it's saying "these are the keys to use for trusting signatures". Could say "public-keys"?

Reply:
Works for me.

Reply:
Ok. If we get to a place of conceptually agreeing I'll make the change :)